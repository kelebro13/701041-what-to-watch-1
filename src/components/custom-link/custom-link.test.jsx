import renderer from "react-test-renderer";
import CustomLink from './custom-link';

it(`render correctly CustomLink componen`, () => {
  const id = `test_button`;
  const className = `test_class`;
  const title = `Test`;
  const handleButtonClick = () => {};

  const tree = renderer
    .create(<CustomLink id={id} className={className} title={title} onClick={handleButtonClick}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
