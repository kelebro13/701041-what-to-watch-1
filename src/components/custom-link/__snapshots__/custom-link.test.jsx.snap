// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly CustomLink componen 1`] = `
<a
  className="test_class"
  id="test_button"
  onClick={[Function]}
>
  Test
</a>
`;
